solr/core/src/test/org/apache/solr/schema/EnumFieldTest.java

```diff
@@ -592,6 +592,11 @@ public void testFacetEnumSearch() throws Exception {
 
     assertQ(req("fl", "" + FIELD_NAME, "q", FIELD_NAME + ":*", "json.facet", jsonFacetParam),
         "//*[@name='facets']/int/text()=6",
-        "//*[@name='allBuckets']/long/text()=6");
+        "//*[@name='allBuckets']/long/text()=6",
```

Comment:
So there was a gap in our testing then, I suppose?

Reply:
Yes, found by a new integration test on our side